XM_Dinner/44_Gloom.yaml

```diff
@@ -19,4 +19,27 @@ shiny: True
 ivs: {hp: 0, atk: 31, def: 0, spA: 31, spD: 31, spe: 31}
 evs: {hp: 0, atk: 0, def: 0, spA: 0, spD: 252, spe: 252}
 nature: Timid
+hidden: false
+---
+species: Gloom
+ingamename: Falaf Bowl
+displayname: Falafel Bowl
+setname: Dinner2
+tags: [Dinner]
+ball: [Cherish]
+item: Miracle Seed
+ability: Battle Armor
+moves:
+    - [Seed Bomb]

```

Comment:
I'd prefer to see some coverage, the concept should allow for a "ball" or "spicy" move of some kind to fit.